File: 프로그래머스/1/12928. 약수의 합/약수의 합.js
function solution(n) {
    var answer = 0;
    for(let i=1;i<=Math.sqrt(n);i++){
        if(n%i==0){
            answer+=i;
            if(i*i!==n)
            answer+=(n/i);
        }
    }
    return answer;
}

File: 프로그래머스/1/12928. 약수의 합/약수의 합.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for(int i=1;i<=(int)Math.sqrt(n);i++){
            if(n%i==0){
                answer+=(i+n/i);
            }
            if(i==Math.sqrt(n)){
                answer-=i;
            }
        }
        return answer;
    }
}